File: app/src/main/java/com/tmvlg/smsreceiver/backend/FreeMessage.java
package com.tmvlg.smsreceiver.backend;

public class FreeMessage {
    public String message_title;
    public String message_text;
    public String time_remained;

    public FreeMessage(String message_title, String message_text, String time_remained) {
        this.message_title = message_title;
        this.message_text = message_text;
        this.time_remained = time_remained;
    }

    public FreeMessage() {

    }
}

File: app/src/main/java/com/tmvlg/smsreceiver/domain/freemessage/FreeMessage.kt
package com.tmvlg.smsreceiver.domain.freemessage

data class FreeMessage(
        val message_title: String,
        val message_text: String,
        val time_remained: String,
        var id: Int = UNDEFINED_ID
) {
    companion object {
        const val UNDEFINED_ID = -1
    }
}